content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -352,102 +352,6 @@ exampleWithKubeConfig = do
-
-  - There are a number of other libraries available, please refer to the [Client Libraries](/docs/reference/using-api/client-libraries/) page.
-
-In each case, the service account credentials of the Pod are used to communicate
-securely with the API server.
-
-#### Directly accessing the REST API
-
-While running in a Pod, the Kubernetes apiserver is accessible via a Service named
-`kubernetes` in the `default` namespace. Therefore, Pods can use the
-`kubernetes.default.svc` hostname to query the API server. Official client libraries
-do this automatically.
-
-The recommended way to authenticate to the API server is with a
-[service account](/docs/tasks/configure-pod-container/configure-service-account/) credential. By default, a Pod
-is associated with a service account, and a credential (token) for that
-service account is placed into the filesystem tree of each container in that Pod,
-at `/var/run/secrets/kubernetes.io/serviceaccount/token`.
-
-If available, a certificate bundle is placed into the filesystem tree of each
-container at `/var/run/secrets/kubernetes.io/serviceaccount/ca.crt`, and should be
-used to verify the serving certificate of the API server.
-
-Finally, the default namespace to be used for namespaced API operations is placed in a file
-at `/var/run/secrets/kubernetes.io/serviceaccount/namespace` in each container.
-
-#### Using kubectl proxy
-
-If you would like to query the API without an official client library, you can run `kubectl proxy`
-as the [command](/docs/tasks/inject-data-application/define-command-argument-container/)
-of a new sidecar container in the Pod. This way, `kubectl proxy` will authenticate
-to the API and expose it on the `localhost` interface of the Pod, so that other containers
-in the Pod can use it directly.
-
-#### Without using a proxy
-
-It is possible to avoid using the kubectl proxy by passing the authentication token
-directly to the API server.  The internal certificate secures the connection.
-
-```shell
-# Point to the internal API server hostname
-APISERVER=https://kubernetes.default.svc
-
-# Path to ServiceAccount token
-SERVICEACCOUNT=/var/run/secrets/kubernetes.io/serviceaccount
-
-# Read this Pod's namespace
-NAMESPACE=$(cat ${SERVICEACCOUNT}/namespace)
-
-# Read the ServiceAccount bearer token
-TOKEN=$(cat ${SERVICEACCOUNT}/token)
-
-# Reference the internal certificate authority (CA)
-CACERT=${SERVICEACCOUNT}/ca.crt
-
-# Explore the API with TOKEN
-curl --cacert ${CACERT} --header "Authorization: Bearer ${TOKEN}" -X GET ${APISERVER}/api
-```
-
-The output will be similar to this:
-
-```json
-{
-  "kind": "APIVersions",
-  "versions": [
-    "v1"
-  ],
-  "serverAddressByClientCIDRs": [
-    {
-      "clientCIDR": "0.0.0.0/0",
-      "serverAddress": "10.0.1.149:443"
-    }
-  ]
-}
-```
-
-
-
+* [Accessing the Kubernetes API from a Pod](/docs/tasks/access-api-from-pod/access-api-from-pod/)
```

Comment:
Thanks @uhari03  for the updates. This links needs to change?

Reply:
Great catch

My bad - can't believe I missed this

Should be fixed now